### ISSUE TITLE
Disabling tracking causes ANR

### ISSUE THREAD

**Issue by kozaxinan** [BranchMetrics/android-branch-deep-linking-attribution#903]
We updated from 5.0.7 from 5.0.4. When we have tracking disabled, app constantly gives ANR after opening new activity.
App visually slow down after calling startActivity. And after some time app gives ANR.

Called methods;
```
disableTracking(true)
removeSessionInitializationDelay()
```

I will include more information if I will be able to collect.

**Comment by echo-branch** [comment]
@bklastaitis-branch 
We'll need to check on this one.

**Comment by kozaxinan** [comment]
Something we noticed this kind of logs after activity change;

```
nput event dispatching timed out sending to a.b.c/a.b.c.FooActvity  Reason: 5cb0be6 a.b.c/a.b.c.FooActvity (server) is not responding. Waited 5001ms for FocusEvent(hasFocus=false)
```

**Comment by marcelpallares** [comment]
I was having the same issue too, it's been resolved in the new v5.0.8

**Comment by kozaxinan** [comment]
I can verify 5.0.8 fixes this issue. But we have different problem with 5.0.8. We will open new issue with details.
